shopfloor_reception_mobile/static/src/scenario/reception.js

```diff
@@ -454,295 +456,17 @@ const Reception = {
             }
             return picking.origin.includes(input);
         },
+        _get_states: function () {
+            return reception_states(this);
```

Comment:
can you try this? See comment below.
```suggestion
            return reception_states.bind(this)();
```

Reply:
Out of curiosity: have you tried this? If it works as I expect, is probably more clear.
Not a blocker though, as we've done the same for checkout_states.

Reply:
i tried it this should work